ci/check_json.py

```diff
@@ -9,10 +9,11 @@
 def _is_json(input_file):
     my_json = json.load(open(input_file,'r'))
     my_key = list(my_json.keys())
-    for key in my_json[my_key[0]].keys():
-        if "stockpile_" in key and "stockpile_output_path" not in key and "stockpile_user" not in key:
-            print(key)
-            print(json.dumps(my_json[my_key[0]][key], indent=4))
+    for i in range(len(my_key)):
+    	for key in my_json[my_key[i]].keys():
+        	if "stockpile_" in key and "stockpile_output_path" not in key and "stockpile_user" not in key:
+                   print(my_key[i],key)
+            	   print(json.dumps(my_json[my_key[i]][key], indent=4))
```

Comment:
Getting this for the non-ocp ci job:
```
  File "ci/check_json.py", line 16
    print(json.dumps(my_json[my_key[i]][key], indent=4))
                                                                                       ^
TabError: inconsistent use of tabs and spaces in indentation
```

Not sure why this is only happening for the stockpile_jjb job and not the new job. That said the spacing does appear off by eyeing it.

Reply:
Fixed it